packages/gnome-shell/src/ui/main.d.ts

```diff
@@ -17,7 +17,7 @@ import { AudioDeviceSelectionDBus } from './audioDeviceSelection.js';
 // import * as InputMethod from '../misc/inputMethod.js'
 // import * as Introspect from '../misc/introspect.js'
 // import * as Keyboard from './keyboard.js';
-// import * as MessageTray from './messageTray.js';
+import * as MessageTray from './messageTray.js';
```

Comment:
I feel bad for nitpicking here, but can you change the import to this (for consistency reasons)?

```suggestion
import { MessageTray } from './messageTray.js';
```

Reply:
Good point. done.